gui/src/components/settings/pages/GeneralSettings.tsx

```diff
@@ -876,6 +884,19 @@ export function GeneralSettings() {
               )}
             />
           </div>
+          <div className="grid sm:grid-cols-2 pb-4">
+            <NumberSelector
+              control={control}
+              name="interface.feedbackSoundVolume"
+              label={l10n.getString(
+                'settings-general-interface-feedback_sound-volume'
+              )}
+              valueLabelFormat={(value) => `${Math.round(value * 100)} %`}
```

Comment:
Can you use ``Intl.NumberFormat`` with  percentage mode? I use it in the Manual Proportions settings